pm-commotion-mac2ip

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash
+#
+# Script written for PittMesh by Jason Khanlar
```

Comment:
If you want to claim copyright on this, put in a header something more like this:

```
© 2016 Jason Khanlar. All rights reserved.
Licensed under the terms of the MIT license.
```

PittMesh isn't an entity for the purposes of licensing. If you want to assign the copyright to @metamesh, that's fine: swap in `Meta Mesh Wireless Communities` for your name and add an authors block that looks like this:

```
AUTHORS
* Jason Khanlar
```